loopd/config.go

```diff
@@ -160,8 +160,8 @@ type Config struct {
 	MaxLogFileSize int    `long:"maxlogfilesize" description:"Maximum logfile size in MB."`
 
 	DebugLevel  string `long:"debuglevel" description:"Logging level for all subsystems {trace, debug, info, warn, error, critical} -- You may also specify <subsystem>=<level>,<subsystem2>=<level>,... to set the log level for individual subsystems -- Use show to list available subsystems"`
-	MaxLSATCost uint32 `long:"maxlsatcost" description:"Maximum cost in satoshis that loopd is going to pay for an LSAT token automatically. Does not include routing fees."`
-	MaxLSATFee  uint32 `long:"maxlsatfee" description:"Maximum routing fee in satoshis that we are willing to pay while paying for an LSAT token."`
+	MaxL402Cost uint32 `long:"maxl402cost" description:"Maximum cost in satoshis that loopd is going to pay for an L402 token automatically. Does not include routing fees."`
```

Comment:
This might cause users that have the old flag set to not be able to start their daemon after upgrading. Should we keep the old flags but set them to `hidden:true` and log a deprecation warning if they are set? Then remove the actual flags in a later release?

Reply:
The flags were re-added in commit "loopd: recorgnize maxlsatcost and maxlsatfee flags".
I ran loopd locally to verify that if both old and new variants of an option are used simultaneously, it fails to start; and if only the old variant is provided, it starts with a deprecation warning.